frappe/desk/search.py

```diff
@@ -74,7 +77,7 @@ def search_widget(doctype, txt, query=None, searchfield=None, start=0,
 	if query and query.split()[0].lower()!="select":
 		# by method
 		try:
-			is_whitelisted(frappe.get_attr(query))
+			query = get_whitelisted_method(query)
```

Comment:
_\<Surprise bug fix\>_ for here and everywhere else `get_whitelisted_method` is used 🥇